src/sbin/sympa

```diff
@@ -0,0 +1,912 @@
+#!/usr/bin/perl
```

Comment:
This shouldn't be included in the repo (`sympa.pl.in` suffices).

Reply:
Sorry, I thought I didn’t added the files, but… my mistake :shrug: